nodebuilder/share/module.go

```diff
@@ -31,6 +30,7 @@ func ConstructModule(tp node.Type, cfg *Config, options ...fx.Option) fx.Option
 		return fx.Module(
 			"share",
 			baseComponents,
+			fx.Invoke(share.EnsureEmptySquareExists),
```

Comment:
Should we still keep this in the base, until we fully migrate to the Store

Reply:
done